tests/performance-tests/atala-performance-tests-k6/README.md

```diff
@@ -1,20 +1,23 @@
-## Prerequisites
+# Prerequisites
 
 - [k6](https://k6.io/docs/getting-started/installation)
 - [NodeJS](https://nodejs.org/en/download/)
 - [Yarn](https://yarnpkg.com/getting-started/install) (optional)
 
-## Installation
+# Installation
 
 **Install dependencies**
 
 Clone the generated repository on your local machine, move to the project root folder and install the dependencies defined in [`package.json`](./package.json)
 
+*NOTE*: The Project has a dependency on `input-output-hk/prism-typescript-client` which is a private repository.
+To install this dependency, you need to have an environment variable `GITHUB_TOKEN` with the scope `read:packages` set, you can install the dependency by running the following command:
```

Comment:
To install this dependency, you need to have an environment variable `GITHUB_TOKEN` with the scope `read:packages` set; you can install the dependency by running the following command: